src/test/kotlin/org/opensearch/replication/integ/rest/ResumeReplicationIT.kt

```diff
@@ -210,6 +210,11 @@ class ResumeReplicationIT: MultiClusterRestTestCase() {
     }
 
     fun `test that replication fails to resume when custom analyser is not present in follower`() {
+
+        if(checkifIntegTestRemote()){
```

Comment:
Can we explore to add @skip annotation based on test configuration?
This can give false sense of execution of tests even though, we don't run these as part of remote cluster configuration.